blob/blob_test.go

```diff
@@ -300,7 +300,6 @@ func TestURLMux(t *testing.T) {
 		name    string
 		url     string
 		wantErr bool
```

Comment:
Note: I added this earlier because I thought that the prefixes were going to be stripped before being passed to the `URLOpener`, but that wasn't easy to do so I ended not doing it; at this point `test.want` == `test.url`.